CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## VNext
 
+- [Removed proactive sampling decision to address high ingestion of telemetry records and item count with adaptive sampling.](https://github.com/microsoft/ApplicationInsights-dotnet/issues/2742)
```

Comment:
Could you highlight that Proactive Sampling was a feature that remained internal and was never used? OR remove changelog to simply say "Fix xyz, which caused incorrect item count when Activity's Recorded flags were modified externally, when enabled side-by-side with OpenTelemetry/other solutions."

Reply:
How about this one

Fixed an adaptive sampling issue that caused incorrect item count when an activity's Recorded flags were modified externally, when enabled side-by-side with OpenTelemetry or other solutions.

Reply:
'Activity'  to make it clear which Activity we are talking about!